tests/cloudformation/checks/resource/aws/example_EC2Credentials/EC2Credentials-FAILED.yaml

```diff
@@ -0,0 +1,19 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Resources:
+  Resource0:
+    Type: AWS::EC2::Instance
+    Properties: 
+      ImageId: ami-04169656fea786776
+      UserData:
+        Fn::Base64:
+          !Sub |
+            #! /bin/bash
+            sudo apt-get update
+            sudo apt-get install -y apache2
+            sudo systemctl start apache2
+            sudo systemctl enable apache2
+            export AWS_ACCESS_KEY_ID
+            export AWS_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
```

Comment:
Thanks @njgibbon . can you add this file to https://github.com/bridgecrewio/checkov/blob/master/.github/exclude-patterns.txt ?

Reply:
@schosterbarak  Just added :)

Reply:
Thank you!